app/javascript/.storybook/storybook.scss

```diff
@@ -7,6 +7,7 @@ body {
     display: grid;
     grid-template-rows: 1fr;
     grid-template-columns: 1fr;
+    padding: 2rem;
```

Comment:
Just so components in the display area aren't stuck to the sides if they are wide components.

Reply:
how about using  `$su-7`? :D

Reply:
I can probably import it. :D This is a Storybook specific SASS file so it is not part of the compiled CSS for the application.

Reply:
gotcha :) up to you :)

Reply:
I'll keep it as just `2rem`, because as mentioned, it really is Storybook specific.